client-app/pages/account/loyalty.vue

```diff
@@ -0,0 +1,200 @@
+                <template #desktop-skeleton>
+                  <tr v-for="i of itemsPerPage" :key="i" class="even:bg-gray-50">
+                    <td class="p-5">
+                      <div class="h-6 bg-gray-200 animate-pulse"></div>
+                    </td>
+                    <td class="p-5">
+                      <div class="h-6 bg-gray-200 animate-pulse"></div>
+                    </td>
+                    <td class="p-5">
+                      <div class="h-6 bg-gray-200 animate-pulse"></div>
+                    </td>
+                    <td class="p-5">
+                      <div class="h-6 bg-gray-200 animate-pulse"></div>
+                    </td>
+                    <td class="p-5">
+                      <div class="h-6 bg-gray-200 animate-pulse"></div>
+                    </td>
+                  </tr>
+                </template>
+                <template #desktop-body>
+                  <tr
+                    v-for="operation in userBalance?.pointsOperations"
+                    :key="operation.id"
+                    class="even:bg-gray-50 hover:bg-gray-200 cursor-pointer"
+                  >
+                    <td class="p-5 overflow-hidden overflow-ellipsis">
+                      {{ moment(operation.createdDate).format("YYYY-MM-DD") }}
+                    </td>
+                    <td class="p-5 overflow-hidden overflow-ellipsis">
+                      {{ operation.reason }}
+                    </td>
+                    <td class="p-5 overflow-hidden overflow-ellipsis">{{ operation.isDeposit }}</td>
+                    <td class="p-5 overflow-hidden overflow-ellipsis">{{ operation.amount }}</td>
+                    <td class="p-5 overflow-hidden overflow-ellipsis">
+                      {{ operation.balance }}
+                    </td>
+                  </tr>
+                </template>
+                <template #desktop-empty>
+                  <!-- Workaround for using colspan -->
+                  <tr>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                    <td></td>
+                  </tr>
+                  <tr>
+                    <td colspan="5" class="polygons-bg">
+                      <div class="flex items-center pl-56 space-x-10 h-80">
+                        <img src="/static/images/account/icons/no-addresses.svg" alt="No operations" />
+                        <div class="flex flex-col space-y-2">
+                          <span class="text-base">There are no operation yet</span>
+                        </div>
+                      </div>
+                    </td>
+                  </tr>
+                </template>
+              </VcTable>
+              <VcInfinityScrollLoader
+                v-if="isInfinityScrollEnable"
+                :loading="loading"
+                distance="100"
+                class="mt-9"
+                @visible="loadMoreOperations()"
```

Comment:
It is not needed to use parenthesis in handlers if you are not passing any additional payload.